src/goto-programs/destructor_tree.cpp

```diff
@@ -0,0 +1,108 @@
+#include "destructor_tree.h"
```

Comment:
Missing "copyright" header

Reply:
Please use the same shape as used in all other files, whether it's useful or not: one day we might want to replace all of them, and that's only possible if the take the same form.